dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include <filesystem>
+#include <fstream>
+
+void Help();
+HRESULT JustDoIt(PCWSTR path) noexcept;
+void AddPackageIfNecessary(PCWSTR path, const std::wstring& filename, const std::wstring& packageFullName);
+bool NeedToRegisterPackage(const std::wstring& packageFullName);
+bool NeedToRegisterPackage(PCWSTR packageFullName);
+HRESULT AddPackage(PCWSTR path, const std::wstring& filename);
+
+class PackageId
```

Comment:
Will get moved to dev\common\AppModel.Package.h in a later PR. Needs some work before it's ready for general use so I didn't want to hold up this work on that cleanup/refactoring